### PR TITLE
Highlight multiple contexts in private images doc [CIRCLE-29510]

### PR DESCRIPTION
**Pull request by lewang**
# Description
Followup to https://github.com/circleci/circleci-docs/pull/4744

# Reasons
Spread the good news.

### CODE REVIEW

File: jekyll/_cci2/private-images.md [hunk 1]
```diff
@@ -17,9 +17,19 @@ Authenticated pulls allow access to private Docker images.  It may also grant hi
...
 
-For the [Docker]({{ site.baseurl }}/2.0/executor-types/#using-docker) executor, specify username and password in the `auth` field of your [config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file.  To protect the password, create a [context]({{ site.baseurl }}/2.0/contexts) or Environment Variable in the CircleCI Project Settings page, and then reference it:
+For the [Docker]({{ site.baseurl }}/2.0/executor-types/#using-docker) executor, specify username and password in the `auth` field of your [config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file.  To protect the password, place it in a [context]({{ site.baseurl }}/2.0/contexts).  Note that CircleCI supports multiple contexts, which is a great way modularize secrets ensuring jobs only access the secrets they need.  Using per-project Environment Variable is another (less flexible) option.
```
Comment (thomasr8) [inline jekyll/_cci2/private-images.md:20]:
```suggestion
For the [Docker]({{ site.baseurl }}/2.0/executor-types/#using-docker) executor, specify username and password in the `auth` field of your [config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file. To protect the password, place it in a [context]({{ site.baseurl }}/2.0/contexts). Note that CircleCI supports multiple contexts, which is a great way modularize secrets ensuring jobs only access the secrets they need. Using per-project Environment Variable is another (less flexible) option.
```

**Review by thomasr8 — APPROVED** [review]
Just a couple of very minor comments about spacing. Otherwise, looks really good.

**Review by glenjamin — APPROVED** [review]
Maybe say something about how having a docker-specific context allows you to add the new context even if your job is already using a context?

File: jekyll/_cci2/private-images.md (hunk 1 above)
Comment (rosieyohannan) [inline jekyll/_cci2/private-images.md:20]:
```suggestion
For the [Docker]({{ site.baseurl }}/2.0/executor-types/#using-docker) executor, specify username and password in the `auth` field of your [config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file.  To protect the password, place it in a [context]({{ site.baseurl }}/2.0/contexts).  

**Note:** CircleCI supports multiple contexts, which is a great way modularize secrets, ensuring jobs only access the secrets they need.  Using per-project Environment Variables is another, less flexible, option.
```

Comment (rosieyohannan) [reply jekyll/_cci2/private-images.md:20]: just a suggestion to draw attention to the note

**Review by melyalex — APPROVED** [review]

**Review by melyalex — APPROVED** [review]

**Pull request merged**